man/Makefile

```diff
@@ -0,0 +1,21 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+SPHINXPROJ    ?= pyenv
+SOURCEDIR     = .
+BUILDDIR      = _build
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
```

Comment:
In most cases you have to install it. Add it to your development requirement.

Reply:
Okay, I was unclear.
Our test makefile installs Bats if it's not already present.
The help makefile should probably do the same for Sphinx.

Reply:
Hmn, so you mean if someone runs `make help` it should automatically install Sphinx? Or do you mean to create another make target (for example `sphinx`) which does that job?

I think amending the `help` target is a bad idea. Installing a Python module/package requires usually a virtual Python environment to separate development from production and to avoid version conflicts with the system. Doing so automatically in the background without the consent of the user is IMHO problematic.

I would suggest to change the root Makefile and add a general `develop` target. We have already the `bats` target. We could amend this root Makefile like this:

```make
.PHONY: sphinx develop
sphinx:
    # create a Python environment local to this Git repo if not available:
    [[ -e .venv ]] || python3 -m venv .venv
    .venv/bin/pip3 install --upgrade pip
    .venv/bin/pip3 install sphinx

# bats target unchanged:

develop: sphinx bat
    @echo "Developing environment prepared."
```

When you start developing, you call these steps:

1. Call `make develop` and everything is done to prepare your environment. 
2. Activate the Python virtual environment with `source .venv/bin/activate`. This can't be done in the Makefile.
3. Build the manpage.

Reply:
Okay, it's unclear how to better do this so far.
We'll need to install Sphinx as part of CI anyway so let's stick with what will show to be convenient for that task.